OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Logic.cs

```diff
@@ -11,6 +11,9 @@
 
 namespace OtripleS.Web.Api.Tests.Unit.Services.CalendarEntryAttachments
 {
+    using System;
+    using System.Linq;
```

Comment:
Why are you putting the usings on the inside of your namespace scope?

Reply:
I have installed code ReSharper and I think it is doing that.